### PR TITLE
BDLaethg35

### PR DESCRIPTION
**Pull request by thea-m**
Hello again,

Here's the next one. I'm aware that not everyone finished looking at b. 6, and I absolutely don't want to make any pressure!! I just thought it's better to put everything that's there out, so that all can arrange their time as is best for them.
Here, I have two issues:

- could you have a look at the pencil note on the inner front board, which I can't read completely?
- there is nothing specific about the history, my palaeographic dating is also very broad. We might discuss in a meeting how to deal with such cases and whether we'll include a history section there.

Thank you and please take your time!!
Dorothea

### CODE REVIEW

File: OxfordBodleian/Juel-Jensen/BDLaethg35.xml [hunk 1]
```diff
@@ -48,11 +153,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p><locus from="ir" to="iiv"/> and have been cut at the bottom.</p>
```
Comment (ElianaDalSasso) [inline OxfordBodleian/Juel-Jensen/BDLaethg35.xml:156]: "and have": is something is missing before "and"?

**Review by ElianaDalSasso — COMMENTED** [review]
Thank you again Thea! Just a doubt, perhaps it is a typo.

File: OxfordBodleian/Juel-Jensen/BDLaethg35.xml (hunk 1 above)
Comment (thea-m) [reply OxfordBodleian/Juel-Jensen/BDLaethg35.xml]: no, it was a mistake, thanks!

**Comment by thea-m** [comment]
@gnizla This is waiting for your review - if you could have a look at it, that would be great (not urgent of course, just noting this now so that it doesn't get lost). Thank you :)

**Comment by gnizla** [comment]
> @gnizla This is waiting for your review - if you could have a look at it, that would be great (not urgent of course, just noting this now so that it doesn't get lost). Thank you :)

I am sorry, I had looked at this a long while ago, I did not have any comments other than thank you so thought I would not add an extra email!

**Pull request merged**